x/gov/client/cli/query.go

```diff
@@ -30,606 +29,20 @@ func GetQueryCmd(ac address.Codec) *cobra.Command {
 	}
 
 	govQueryCmd.AddCommand(
-		GetCmdQueryProposal(),
-		GetCmdQueryProposals(ac),
-		GetCmdQueryVote(ac),
-		GetCmdQueryVotes(),
-		GetCmdQueryParams(),
-		GetCmdQueryParam(),
 		GetCmdQueryProposer(),
```

Comment:
Reason for this staying?

Reply:
Yeah, it queries events and filters (https://github.com/cosmos/cosmos-sdk/blob/90b390795e1b3dd18b871aec589c9f21dbaa4ca2/x/gov/client/utils/query.go#L165). There is no gRPC endpoint that does that in gov.
In this case, we cannot use AutoCLI. I'll implement https://github.com/cosmos/cosmos-sdk/issues/16836, to be able to wire it back in gov.

Reply:
Now that proposers are in the proposals, we could actually have a gRPC endpoints for it too.